libsolidity/CompilerUtils.h

```diff
@@ -30,6 +30,9 @@ namespace solidity {
 
 class Type; // forward
 
+/// The size in bytes of the function (hash) identifier
+static const unsigned int g_functionIdentifierOffset = 4;
```

Comment:
Ok but I think it's quite readable like this too. Plus it's kind of annoying to have static const int in a class if I remember well.